docs/guide/faq/README.md

```diff
@@ -6,6 +6,19 @@ sidebarDepth: 0
 
 [[toc]]
 
+
+##  General limitations that apply to all Zigbee implementations
+
+Note that each Zigbee2MQTT installation instance only supports connecting a single dedicated Zigbee Coordinator radio adapter or module with a single Zigbee network and that the Zigbee Coordinator cannot already be connected or used by any other application. Any devices that are or have previously been connected to another Zigbee implementation will also need to first be reset to their factory default settings before they can be paired/joined to Zigbee2MQTT, please see each device manufacturer's documentation.
+
+Any Zigbee device is limited to only be paired/joined to one Zigbee Coordinator and only be part of one Zigbee network, meaning that Zigbee devices can only be connected to a single Zigbee gateway. If you want to move a Zigbee device to a different Zigbee network then you need to factory reset that Zigbee device and re-pair/re-join it to the other Zigbee Gateway. This is a limitation in the current (as well as previous) Zigbee protocol specifications, governed by the [CSA (Connectivity Standards Alliance)](https://csa-iot.org/all-solutions/zigbee/), and as such, that limitation applies to all Zigbee implementations, not just the Zigbee2MQTT implementation. This also makes the Zigbee Coordinator a SPOF (Single-Point-Of-Failure) in your Zigbee network, so it needs to be available for your Zigbee devices to stay connected.
```

Comment:
> This also makes the Zigbee Coordinator a SPOF (Single-Point-Of-Failure) in your Zigbee network, so it needs to be available for your Zigbee devices to stay connected.

I do not fully agree with this, the Zigbee devices stay connected even when the coordinator is down (a mesh). If the coordinator goes down, you of course cannot control them via the coordinator anymore but devices can still control each other through binding.

Reply:
@Koenkk OK removed that line now.